bloom_nofos/nofos/api/tests.py

```diff
@@ -0,0 +1,140 @@
+from django.test import TestCase, override_settings
+from django.conf import settings
+from nofos.models import Nofo, Section, Subsection
+import json
+import os
+
+
+@override_settings(API_TOKEN="test-token-for-ci")
+class NofoAPITest(TestCase):
+    def setUp(self):
+        self.valid_token = "test-token-for-ci"
+        self.headers = {
+            "HTTP_AUTHORIZATION": f"Bearer {self.valid_token}",
+        }
+
+        # Create test NOFO for export tests
+        self.nofo = Nofo.objects.create(
+            title="API Test NOFO",
+            number="00000",
+            tagline="Test me via API!",
+            theme="landscape-cdc-blue",
+            group="bloom",
+        )
+        self.section = Section.objects.create(
+            nofo=self.nofo, name="API Test NOFO: Section 1", order=1
+        )
+        self.subsection = Subsection.objects.create(
+            section=self.section,
+            name="API Test NOFO: Subsection 1",
+            order=1,
+            tag="h3",
+        )
+
+        # Load fixture data for import tests
+        fixture_path = os.path.join(
+            settings.BASE_DIR, "nofos", "fixtures", "json", "cms-u2u-25-001.json"
+        )
```

Comment:
Great! 

There are a few other fixtures in there I used for making sure bad data was overwritten (`hrsa-` and `published-`), but I think we can include those checks in the `import_nofo` test and then just remove these other fixtures.